src/js/components/AppPageComponent.jsx

```diff
@@ -343,11 +343,8 @@ var AppPageComponent = React.createClass({
     var volumes = AppsStore.getVolumes(model.id);
 
     var tabs = state.tabs.filter(tab =>
-      tab.text !== "Volumes" || (
-        model.container != null &&
-        model.container.volumes != null &&
-        model.container.volumes.length > 0
-      ));
```

Comment:
@pierlo-upitup That's true in that case so if there aren't any tasks present in the JSON we don't need the Volumes tab.